zerver/webhooks/netlify/doc.md

```diff
@@ -0,0 +1,18 @@
+Get Zulip notifications for your Netlify deployments!
+
+1. {!create-stream.md!}
+
+2. {!create-bot-construct-url-indented.md!}
+
+3. Go to your Netlify project, and click **Settings**. Click **Build & deploy**, and select **Deploy notifications**.
+   Click **Add Notification**, and select **Outgoing webhook**.
+
+4. Select an **Event**, and set **URL to notify** to the URL constructed above. Click **Save**.
+
+{!congrats.md!}
+
+![](/static/images/integrations/netlify/001.png)
+
+!!! tip "" For more information regarding Netlify webhooks, see:
```

Comment:
This isn't getting rendered correctly! I'd recommend looking at other docs and then replicating exactly what they do for `tip` sections! :)